operate/client/src/App/Processes/ListView/v2/Filters/index.tsx

```diff
@@ -47,12 +47,7 @@ import {
   splitDefinitionIdentifier,
 } from 'modules/hooks/processDefinitions';
 
-interface FilterValues extends ProcessInstancesFilter {
-  variableName?: string;
-  variableValues?: string;
-}
```

Comment:
I noticed that a recent change added these values to the `ProcessInstancesFilter` in _utils/filter/v2/processInstancesSearch_. So I quickly cleaned it up here as well...